test/post-911-gib-status/components/UserInfoSection.unit.spec.js

```diff
@@ -1,19 +1,52 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
+import _ from 'lodash';
 
 import UserInfoSection from '../../../src/js/post-911-gib-status/components/UserInfoSection.jsx';
 
 const props = {
   enrollmentData: {
   }
 };
+const currentHeadingSelector = '#up-to-date-header';
 
 describe('<UserInfoSection>', () => {
   it('should render', () => {
     const tree = SkinDeep.shallowRender(<UserInfoSection {...props}/>);
     const vdom = tree.getRenderOutput();
     expect(vdom).to.not.be.undefined;
   });
-  // maybe test print button does something
+
+  describe('props.showCurrentAsOfAlert=false', () => {
```

Comment:
I'm not really sure what the convention is for grouping, but in most of the vets.gov unit tests I've seen, we throw tests into just 1 group per component (sometimes 2) and give the tests slightly more descriptive names, like
```
it('should display warning'
it('should not display warning')
it('should not display warning when showCurrentAsOfAlert is not passed in')
```
I think your grouping might actually be a better reflection of what mocha was defined for, i.e. the `describe` says something about the state and it's `it` statements each test something that should be true about that state. Though I think the describe statements should generally be natural language, not code snippets.

Maybe these can be 2 groups:
```
describe('showCurrentAsOfAlert is falsey')
```
with  both tests inside, and
```
describe('showCurrentAsOfAlert is truthy')
```
with just one test.